frontend/src/modules/Organizations/components/OrganizationTeamInviteEditForm.js

```diff
@@ -104,5 +209,6 @@ OrganizationTeamInviteEditForm.propTypes = {
   team: PropTypes.object.isRequired,
   onClose: PropTypes.func,
   reloadTeams: PropTypes.func,
-  open: PropTypes.bool.isRequired
+  open: PropTypes.bool.isRequired,
+  allPermissions: PropTypes.any
```

Comment:
Not sure if we need to include enqueueSnackbar as a needed prop

Reply:
In most cases in our code it's not included.